client/src/app/ui/modules/list/components/list/list.component.ts

```diff
@@ -560,11 +560,11 @@ export class ListComponent<V extends Identifiable> implements OnInit, OnDestroy
                 return true;
             }
 
-            // filter by ID
+            // filter by sequential number
             const trimmedInput = this.inputValue.trim().toLowerCase();
-            const idString = `` + item.id;
```

Comment:
Pay attention here: Keep in mind, that there are models without a `sequential_number`